lib/cs_guide_web/router.ex

```diff
@@ -35,6 +35,7 @@ defmodule CsGuideWeb.Router do
     resources("/sessions", SessionController, only: [:new, :create])
 
     get("/json_drinks", DrinkController, :json_index)
+    get("/:page_name", StaticPageController, :show)
```

Comment:
This is actually going to have to go in it's own scope block underneath everything. Otherwise a call to `/admin` will be interpreted as a static page, instead of going to the admin scope. Just put this at the bottom:
``` elixir
scope "/search", CsGuideWeb do
  pipe_through(:browser)

  get("/:page_name", StaticPageController, :show)
end
```